src/components/Class/index.js

```diff
@@ -167,6 +169,15 @@ function Class({ classToEdit, indicator }) {
     });
   };
 
+  const convertToIST = (d) => {
+    const b = d.split(/\D+/);
+    const dateInObj = new Date(
+      Date.UTC(b[0], --b[1], b[2], b[3], b[4], b[5], b[6])
+    );
+    const utc = dateInObj.getTime() + dateInObj.getTimezoneOffset() * 60000;
```

Comment:
Assuming this file is in the front end code and running in browser,
does  dateInObj.getTimezoneOffset() return offset of this time to UTC? 
Is utc in millis here?